.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
-.vscode
\ No newline at end of file
+.vscode
+lib
```

Comment:
was this added automatically?

Reply:
@giladgurvich no, I let IJ download the groovy jdk for syntax highlighting and it added a bunch of files to this directory, so I added it to gitignore